_drafts/circuit-simulator-blog.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Circuit Simulator: What does 'game' even mean?"
+date:   2019-02-02 12:18:11 -0500
+category: blog
+---
+4me/
+
+> What do I want to say?
+
+> What voice do I use? what perspective??
+
+> I want to give the history of the game. Why it's important to me and why you might enjoy it more than just playing it.
+
+### Inspiration
+
+Sam and Aaron picked it up in October of 2015. They had each worked on a few small games, and had planned ideas together, but this was the first parallel coding exercise. They decided to tackle this with the most vanilla JS that they could do, recently having learned that scoping a project properly will have a major impact. You can see the origins of these blossoming, young professional devs [in the CircuitGame Repo](https://github.com/SylverStudios/CircuitGame).
+
+Aaron had always been excited about solving circuit diagrams in school and thought it would be a fun game to be interactive, versus pen and paper. Boy was he right! The first pass at circuit simulator was not only impossible, but it was completely devoid of fun. They called it a smashing success.
+
+<!-- I think there needs to be a little section talking about what the game actually is -->
+
+### Development
+
+The development was the most exciting part of this project for the team. Much white boarding and sitting on the couch together coding ensuied. The really exciting part of developmentwas that each circuit was a Directed Asyclic Graph, which meant Sam and Aaron got to write some fun recursive functions. Also there was a backend model to generate and test a graph for success, this was Sam's domain. Whipped up with a passion that would frighten most adults, Sam produced a great genatation engine. At the same time, Aaron was working on UI generation tool that could traverse the graphs that Sam produced and render a nice little interactive circuit diagram.
```

Comment:
> sitting on the couch together coding ensuied
"ensuied" misspelled D: also this may be less awkwardly worded like "coding on the couch together"

Reply:
> Asyclic

Acyclic

Reply:
"genetation"... generation?

Reply:
🤦‍♂️ ty

Reply:
what's up with spell check not kicking in…